tests/fsharp/update.base.line.with.actuals.fsx

```diff
@@ -1,27 +0,0 @@
-open System.IO
```

Comment:
there was a pre-existing script being more accurate, updated the readme.md as well.